tgui/packages/tgui/interfaces/ExplosiveCompressor.js

```diff
@@ -153,13 +161,13 @@ const ExplosiveCompressorContent = (props, context) => {
     </Section>
   );
   return (
-    <>
-      <Stack.Item grow>{CoreBreakdown}</Stack.Item>
-      <Stack.Item grow>{BombBreakdown}</Stack.Item>
-    </>
+    [
+      <Stack.Item grow key="core">{CoreBreakdown}</Stack.Item>,
+      <Stack.Item grow key="ttv">{BombBreakdown}</Stack.Item>,
+    ]
```

Comment:
You should revert this, fragments were designed specifically for this usecase.

Reply:
Oh, i didn't know that. Will do 👍